x-pack/plugins/fleet/common/constants/file_storage.ts

```diff
@@ -8,26 +8,26 @@
 // File storage indexes supporting file upload from the host to Elastic/Kibana
 // If needing to get an integration specific index name, use the utility functions
 // found in `common/services/file_storage`
-export const FILE_STORAGE_METADATA_INDEX_PATTERN = '.fleet-files-*';
-export const FILE_STORAGE_DATA_INDEX_PATTERN = '.fleet-file-data-*';
+export const FILE_STORAGE_METADATA_INDEX_PATTERN = '.fleet-fileds-fromhost-meta-*';
+export const FILE_STORAGE_DATA_INDEX_PATTERN = '.fleet-fileds-fromhost-data-*';
 
-// File storage indexes supporting user uplaoded files (via kibana) that will be
+// File storage indexes supporting user uploaded files (via kibana) that will be
```

Comment:
😂

Reply:
I know I know. and: my IDE has spell check too 🤦